tests/test_edgeql_ddl.py

```diff
@@ -10270,6 +10270,244 @@ async def test_edgeql_ddl_captured_as_migration_01(self):
             }]
         )
 
+    async def test_edgeql_ddl_link_policy_01(self):
```

Comment:
Let's add some tests with constraints like case in #2232 to test that we're clearing up inline links.  I also have a feeling that `required multi` links are probably also broken with respect to this.

Reply:
I added a test for #2232, and then some broken ones for MULTI.

What is the story with the multi issue? We can't have our constraints operating over multiple tables, so we do the checking on the query side, which means our deletion trigger won't set off the error? Should we just disallow that feature combination?

Reply:
Or maybe we can make the ON DELETE ALLOW specifically understand how to handle required